pkg/action/cl001/ac000/plan.go

```diff
@@ -36,17 +36,17 @@ var Plan = []plan.Step{
 	},
 	{
 		Action:  "ac004",
-		Backoff: plan.NewBackoff(24*time.Minute, 3*time.Minute),
+		Backoff: plan.NewBackoff(30*time.Minute, 3*time.Minute),
```

Comment:
Here it makes sense to wait for the workers of the node pool to be up. Though half an hour might be intense. What was your reasoning behind that decision?

Reply:
Basically I was assuming the same time as we would have for the master nodes, because we don't create them in the same action. Shall we leave it as it is, 24 min?

Reply:
Don't know what is right or wrong. Let's go with this. Just wanted to understand your decision making process. ❤️